src/components/Tooltip/Tooltip.module.scss

```diff
@@ -108,13 +108,19 @@
     opacity: 0;
     visibility: hidden;
   }
+
+  &.softEdges {
+    border-radius: 8px;
```

Comment:
Why define this twice instead of making just one `.softEdges` as a top level class?

nit: I didn't really know what "softEdges" meant until I read this line.
may I suggest using "addBorderRadius" or "roundedCorners" or "softCorners"